docs/03-explore.md

```diff
@@ -60,19 +67,32 @@ ip-10-0-168-43.ec2.internal    Ready     master    23h       v1.12.4+5dc94f3fda
 ip-10-0-171-135.ec2.internal   Ready     worker    23h       v1.12.4+5dc94f3fda   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/instance-type=m4.large,beta.kubernetes.io/os=linux,failure-domain.beta.kubernetes.io/region=us-east-1,failure-domain.beta.kubernetes.io/zone=us-east-1c,kubernetes.io/hostname=ip-10-0-171-135,node-role.kubernetes.io/worker=
 ~~~
 
-For reference, **labels** are used as a mechanism to tag certain information onto a node, or a set of nodes, that can help you identify your systems, e.g. by operating system, system architecture, specification, location of the system (e.g. region), it's hostname, etc. They can also help with application scheduling, e.g. make sure that my application (or pod) resides on a specific system type. The labels shown above are utilising the default labels, but it's possible to set some custom labels in the form of a key-value pair.
+For reference, **labels** are used as a mechanism to tag certain information
+onto a node, or a set of nodes, that can help you identify your systems, e.g.
+by operating system, system architecture, specification, location of the system
+(e.g. region), it's hostname, etc. They can also help with application
+scheduling, e.g. make sure that my application (or pod) resides on a specific
+system type. The labels shown above are utilising the default labels, but it's
+possible to set some custom labels in the form of a key-value pair.
 
 ## The Cluster Operator
 
-The Cluster Operator is the most important implementation within OpenShift, it's a self-hosted operator that is heavily responsible for installation, management, maintenance, and automated operations on the OpenShift cluster; it's essentially what turns Kubernetes into OpenShift - it enables all of the additional operators that we rely on (including the web console, the marketplace, image registry, and router service, etc) and configures the cluster as we have specified. We can view it's status by using the following command-
+The Cluster Operator is the most important implementation within OpenShift, it's
+a self-hosted operator that is heavily responsible for installation, management,
+maintenance, and automated operations on the OpenShift cluster; it's essentially
+what turns Kubernetes into OpenShift - it enables all of the additional
+operators that we rely on (including the web console, the marketplace, image
+registry, and router service, etc) and configures the cluster as we have
+specified. We can view it's status by using the following command-
 
 ~~~bash
 $ oc get deployments -n openshift-cluster-version
```

Comment:
This only indicates whether the pod has been deployed.  I think there are two more relevant commands.  The first is `oc get clusterversion`, which will output the current version of the OpenShift cluster and indicate a (very) high level status:

```console
[you@bastion]$ oc get clusterversion
NAME      VERSION     AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.0.0-0.7   True        False         28h     Cluster version is 4.0.0-0.7
```

The second command is `oc get clusterversion`, which provides status for the OpenShift cluster operators:

```console
[you@bastion]$ oc get clusteroperator
NAME                                  VERSION   AVAILABLE   PROGRESSING   FAILING   SINCE
cluster-autoscaler                              True        False         False     29h
cluster-storage-operator                        True        False         False     29h
console                                         True        False         False     28h
dns                                             True        False         False     29h
image-registry                                  True        False         False     29h
ingress                                         True        False         False     28h
kube-apiserver                                  True        False         False     29h
kube-controller-manager                         True        False         False     29h
kube-scheduler                                  True        False         False     29h
machine-api                                     True        False         False     29h
machine-config                                  True        False         False     17h
marketplace-operator                            True        False         False     29h
monitoring                                      True        False         False     80m
network                                         True        False         False     81m
node-tuning                                     True        False         False     28h
openshift-apiserver                             True        False         False     81m
openshift-authentication                        True        False         False     29h
openshift-cloud-credential-operator             True        False         False     29h
openshift-controller-manager                    True        False         False     29h
openshift-samples                               True        False         False     29h
operator-lifecycle-manager                      True        False         False     29h
```

Reply:
Thank you!